drivers/couchbase/couchbase-driver/src/main/java/io/mongock/driver/couchbase/repository/ChangeEntryKeyGenerator.java

```diff
@@ -0,0 +1,17 @@
+package io.mongock.driver.couchbase.repository;
+
+import io.mongock.driver.api.entry.ChangeEntry;
+
+public class ChangeEntryKeyGenerator {
+
+  public static final String KEY_PREFIX = "mongock-";
```

Comment:
We suggest to use "mongockChangeEntry-" prefix for ChangeEntry and to use "mongockLock-" prefix for LockEntry. (It's needed to add a LockEntryKeyGenerator for LockEntry).

Reply:
ChangeEntry is ok, but LockEntry hasn't prefix. Currently Lock entries are created with "DEFAULT_KEY" value (it's Mongock's default value) as Couchbase document key, and we think the Couchbase document key should be something like "mongockLockEntry-DEFAULT_KEY", to prevent conflicts with users documents.